dolphin/task_manager/tasks/task.py

```diff
@@ -54,7 +56,18 @@ def sync(self):
             LOG.info("Syncing storage successful!!!")
 
     def remove(self):
-        pass
+        LOG.info('Remove storage device for storage id:{0}'.format(self.storage_id))
+        try:
+            db.storage_delete(self.context, self.storage_id)
```

Comment:
should concern about access_info and alert_source

Reply:
Access_info has been involved

Reply:
Done